exec/java-exec/src/main/java/org/apache/drill/exec/record/metadata/RepeatedListBuilder.java

```diff
@@ -26,12 +26,15 @@
  * here. It would certainly be cleaner to have a single field, with the
  * number of dimensions as a property, but that is not how Drill evolved.
  */
-
 public class RepeatedListBuilder implements SchemaContainer {
 
   private final SchemaContainer parent;
   private final String name;
-  private AbstractColumnMetadata child;
+  private ColumnMetadata child;
+
+  public RepeatedListBuilder(String name) {
```

Comment:
Here and elsewhere. Since these classes now work in two modes, might be good to explain that somewhere, especially how to use the class as a standalone builder.